processor/cumulativetodeltaprocessor/README.md

```diff
@@ -72,3 +74,6 @@ processors:
         # If include/exclude are not specified
         # convert all cumulative sum metrics to delta
 ```
+
+[beta]: https://github.com/open-telemetry/opentelemetry-collector-contrib#beta
```

Comment:
This link was wrong @TylerHelmuth, I think it should have been:

```
https://github.com/open-telemetry/opentelemetry-collector#beta
```

Reply:
Sure thing! Opened an issue https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/10162